pkgs/applications/editors/jupyter-kernels/coq/default.nix

```diff
@@ -0,0 +1,67 @@
+{ lib
+, stdenv
+, callPackage
+, runCommand
+, makeWrapper
+, coq
+, imagemagick
+, python3
+}:
+
+# To test (in root nixpkgs dir):
+# $(nix-build -E 'with import ./. {}; jupyter.override { definitions = { coq = coq-kernel.definition; }; }')/bin/jupyter-notebook
+
+# To test with packages:
+# $(nix-build -E 'with import ./. {}; jupyter.override { definitions = { coq = coq-kernel.definitionWithPackages [coqPackages.ceres]; }; }')/bin/jupyter-notebook
+
+let
+  kernel = callPackage ./kernel.nix {};
+
+in
+
+rec {
+  launcher = runCommand "coq-kernel-launcher" {
+    inherit coq;
```

Comment:
```suggestion
```

Reply:
I swear I was told the opposite once in a previous review :)